### ISSUE TITLE
ImageDataGenerator().flow_from_directory to make an AutoEncoder

### ISSUE THREAD

**Issue by AryanSethi** [mjkvaak/ImageDataAugmentor#6]
I'm trying to make an AutoEncoder using the keras ImageDataGenerator function with 'flow_from_directory()'. Now because I want to build an Autoencoder, I dont require the image labels because the Y_train of my dataset are not labels but images itself.
When I try to fit my model and pass X_train and Y_train it shows an error :

y argument is not supported when using keras.utils.Sequence as input.

How can I get away with this.
